lib/fathead/git_manual/parse.py

```diff
@@ -16,24 +16,27 @@ class Command(object):
     def __init__(self, name, description, filename):
         '''Instantiate the information about the command'''
         self.name = name
-        self.description = description
+        self.description = description.replace('\n', '\\n').replace('\t', '    ')
         self.filename = filename
         self.usage = ''
 
-    def __str__(self):
+    def basic_usage(self):
         '''
         Output the git command information in the proper format required for
         DuckDuckGo Fatheads
         '''
-
         # Clean up the usage statement which can have newline characters and
         # tab characters, which mess up how it renders
         usage_cleaned = self.usage.replace('\n', '\\n').replace('\t', '    ')
 
-        # Make the abstract have the description as well as a code block
-        # for the usage of the command
-        abstract = '{}\n<pre><code>{}</pre></code>'.format(self.description,
-                                                           usage_cleaned)
+        if usage_cleaned:
+            # Make the abstract have the description as well as a code block
+            # for the usage of the command
+            abstract = '{}\n<pre><code>{}</pre></code>'.format(self.description,
```

Comment:
@moollaza ah sorry about this -- I thought you had only wanted to strip newline characters from new code related to this feature -- I didn't realize this was causing problems in production as well for the basic git commands